PWGCF/DataModel/FemtoDerived.h

```diff
@@ -50,6 +51,9 @@ DECLARE_SOA_COLUMN(BitMaskTrackTwo, bitmaskTrackTwo, BitMaskType);     //! Bit f
 DECLARE_SOA_COLUMN(BitMaskTrackThree, bitmaskTrackThree, BitMaskType); //! Bit for track three
 
 DECLARE_SOA_COLUMN(Downsample, downsample, bool); //! Flag for downsampling
+
+DECLARE_SOA_COLUMN(QnVal, qnVal, int);         //! qn bins for dividing events
+DECLARE_SOA_COLUMN(Occupancy, occupancy, int); //! TPC occupancy
```

Comment:
I would not stay TPC occupancy in the comment
Occupancy might come from other estimators